client/my-sites/marketplace/pages/marketplace-plugin-install/index.tsx

```diff
@@ -123,15 +124,9 @@ const MarketplacePluginInstall = ( {
 	);
 	const isJetpackSelfHosted = selectedSite && isJetpack && ! isAtomic;
 
-	const supportsAtomicUpgrade = useRef< boolean >();
-	useEffect( () => {
-		supportsAtomicUpgrade.current =
-			selectedSite?.plan &&
-			( isPro( selectedSite.plan ) ||
-				isBusiness( selectedSite.plan ) ||
-				isEnterprise( selectedSite.plan ) ||
-				isEcommerce( selectedSite.plan ) );
-	}, [ selectedSite ] );
+	const supportsAtomicUpgrade = useSelector( ( state ) =>
```

Comment:
Should this be `hasAtomicFeature`? `supportsAtomicUpgrade` might be confusing as sites already have the feature and don't actually support an upgrade.